handlers/aws/handler.py

```diff
@@ -141,6 +141,7 @@ def lambda_handler(lambda_event: dict[str, Any], lambda_context: context_.Contex
 
     if trigger_type == "cloudwatch-logs":
         cloudwatch_logs_event = _from_awslogs_data_to_event(lambda_event["awslogs"]["data"])
+        lambda_arn = parse_arn(lambda_context.invoked_function_arn)
```

Comment:
Why do `parse_arn`?

We could just remove this line and the new function and do:

```python
region = os.environ['AWS_REGION']
```

[Here](https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html#configuration-envvars-runtime) are the lambda enviroment variables.

Reply:
I am not a fan of looking up environment variables in the middle of business code. I usually prefer to look them up during application initialization.

I see many uses of the ARN namedtouple, but I you're right: let's keep this simple.

Reply:
Yeah, I see. 

I', adding a helper function to deault with AWS_REGION and AWS_DEFAULT_REGION.

Reply:
Switched lambda region look up to environment variables.